targets/TARGET_Maxim/TARGET_MAX32620C/device/TOOLCHAIN_IAR/MAX32620.icf

```diff
@@ -16,7 +16,7 @@ define region RAM_region      = mem:[from __region_RAM_start__   to __region_RAM
 
 /* Stack and Heap */
 define symbol __size_cstack__ = 0x1000;
-define symbol __size_heap__   = 0x4000;
+define symbol __size_heap__   = 0xF000;
```

Comment:
That is quite the jump in heap allocation.

Though it looks like this target specific, so that should be fine.